site/content/docs/4.3/content/tables.md

```diff
@@ -285,7 +358,43 @@ Add `.table-borderless` for a table without borders.
     </tr>
     <tr>
       <th scope="row">3</th>
-      <td colspan="2">Larry the Bird</td>
+      <td>Larry</td>
+      <td>the Bird</td>
+      <td>@twitter</td>
+    </tr>
+  </tbody>
+</table>
+{{< /example >}}
+
+And also works with table color classes:
+
+{{< example >}}
+<table class="table table-striped">
```

Comment:
This table looks pretty ugly 😄

Reply:
Yeah I know lol... remove it? Find a better example?